combadge/Cargo.toml

```diff
@@ -9,3 +9,6 @@ structopt = { version = "0.2" }
 byteorder = "*"
 walkdir = "*"
 flate2 = "*"
+log = "*"
+env_logger = "*"
+custom_error = "*"
```

Comment:
I suggest you use `thiserror` instead, it's up to you.
It provides cleaner code, and is a bit better maintaned.
https://crates.io/crates/thiserror
https://docs.rs/thiserror/1.0.13/thiserror/